src/appConfig.js

```diff
@@ -410,7 +410,47 @@ configuration.regions = [
                         "label": "Great River (GR)"
                     }]
                 }],
-
+            },
+            "SedimentRegions": {
```

Comment:
![image](https://github.com/USGS-WiM/StreamStats/assets/40237491/ac66d88b-b231-4c96-9624-f5e38830eb2d)
 I would recommend changing this popup to:

**MN Sediment Regions**
Sediment Region: Southwest
Region Abbreviation: SW

Reply:
Thanks for catching that! I was thinking if I didn't have the query properties there wouldn't be a popup at all.